pkg/network/deviceinfo/sriov.go

```diff
@@ -57,45 +54,26 @@ func CreateNetworkPCIAnnotationValue(networks []v1.Network, interfaces []v1.Inte
 
 func mapNetworkNameToPCIAddress(networks []v1.Network, interfaces []v1.Interface,
 	networkStatusAnnotationValue string) (map[string]string, error) {
-	multusInterfaceNameToNetworkStatusMap, err := mapMultusInterfaceNameToNetworkStatus(networkStatusAnnotationValue)
+	networkNameToDeviceInfo, err := mapNetworkNameToDeviceInfo(networks, interfaces, networkStatusAnnotationValue)
 	if err != nil {
 		return nil, err
 	}
-	networkNameScheme := namescheme.CreateNetworkNameSchemeByPodNetworkStatus(networks, multusInterfaceNameToNetworkStatusMap)
 
 	networkPCIMap := map[string]string{}
 	for _, sriovIface := range vmispec.FilterSRIOVInterfaces(interfaces) {
-		multusInterfaceName := networkNameScheme[sriovIface.Name]
-		networkStatusEntry, exist := multusInterfaceNameToNetworkStatusMap[multusInterfaceName]
```

Comment:
nit: It took me some time to understand, maybe if it was in a separated commit it could be easier to understand.
Not sure its possible though.

Reply:
Unless you insist I prefer to keep it in this commit, I think it shows better the use case of the new common method.